### DISCUSSION TITLE
Master to Deatails

### DISCUSSION THREAD

**Discussion by lotffy03** [serenity-is/Serenity#7248] [Help]
how to create master to detail

**Comment by L30P1N0** [comment]
Check this

https://serenity.is/docs/tutorials/movies/10-keeping-record-of-the-cast

**Comment by LSNicholls** [comment]
Also look at this https://demo.serenity.is/BasicSamples/FilteredLookupInDetailDialog which IMHO is a more straightforward (maybe newer?) approach.  You don't have to be interested in the "filtered category" lookup aspect of the sample ; the important thing to notice is that you are looking at an order and its details.  

This example is a master form with a grid of the details.  If, instead, you want a master grid and a detail grid, you can do this similarly to this example. although I don't know if there is an available sample in the demo.
